### PR TITLE
Add autoCapitalize prop

### PR DESCRIPTION
**Pull request by Shmew**
https://developer.mozilla.org/en-US/docs/Web/HTML/Global_attributes/autocapitalize

### CODE REVIEW

File: Feliz/Properties.fs [hunk 1]
```diff
@@ -1453,3 +1454,14 @@ module prop =
         static member inline url = Interop.mkAttr "type" "url"
         /// Defines a week and year control (no timezone)
         static member inline week = Interop.mkAttr "type" "week"
+
+    [<Erase>]
+    type autoCapitalize =
```
Comment (cmeeren) [inline Feliz/Properties.fs]: Should be lowercase: `autocapitalize`

File: Feliz/Properties.fs [hunk 2]
```diff
@@ -1453,3 +1454,14 @@ module prop =
...
+        /// No autocapitalization is applied (all letters default to lowercase)
+        static member inline off = Interop.mkAttr "autoCapitalize" "off"
```
Comment (cmeeren) [inline Feliz/Properties.fs]: All of these should be lowercase: `autocapitalize`

Comment (Shmew) [reply Feliz/Properties.fs]: I thought so too at first, but then I saw how other props in this file use the same casing and it works as expected. Such as [this property](https://github.com/Shmew/Feliz/blob/patch-1/Feliz/Properties.fs#L349). I tested it out and it does apply the attribute in all lower case so I figured I should follow the style of the rest of the file.

Comment (cmeeren) [reply Feliz/Properties.fs]: Oh, right. As long as it works :)

**Pull request merged**